### ISSUE TITLE
IIDIsotropicGaussianUVLoss mistake

### ISSUE THREAD

**Issue by asanakoy** [facebookresearch/detectron2#2192]
Hi guys,

I think there is a small error in
 `loss = 0.5 * (self.log2pi + 2 * torch.log(sigma2) + delta_t_delta / sigma2)` [projects/DensePose/densepose/modeling/losses/chart_with_confidences.py#L158](https://github.com/facebookresearch/detectron2/blob/7007c4e85be451aa2d60e05be8b5ea983cfaf8cf/projects/DensePose/densepose/modeling/losses/chart_with_confidences.py#L158
). 
The constant multiplier before `torch.log(sigma2)` must be **1, not 2**.

This line is equivalent to `loss = -log (1 / (2Pi sigma2**2)**0.5 * Exp(delta_t_delta / sigma2)`.
Since sigma2 is already a sigma squared, then there is a wrong constant 2 before  torch.log(sigma2).
The correct variant is  `loss = -log (1 / (2Pi sigma2)**0.5 * Exp(delta_t_delta / sigma2)` which is  `loss = 0.5 * (self.log2pi + torch.log(sigma2) + delta_t_delta / sigma2)` in code.

**Comment by asanakoy** [comment]
@vkhalidov Please have a look.

**Comment by asanakoy** [comment]
It's alright. Sorry, I missed that `delta_t_delta` is already a sum of squared errors for both `u` and `v`.  Can be closed now :)
